src/bp-templates/bp-nouveau/includes/document/functions.php

```diff
@@ -93,7 +93,7 @@ function bp_nouveau_document_localize_scripts( $params = array() ) {
 		'create_folder_error_title'       => __( 'Please enter title of folder', 'buddyboss' ),
 		'invalid_file_type'               => __( 'Unable to upload the file', 'buddyboss' ),
 		'document_select_error'           => __( 'Please upload only the following file types: ', 'buddyboss' ) . '<br /><div class="bb-allowed-file-types">' . implode( ', ', array_unique( $extensions ) ) . '</div>',
-		'dropzone_document_message'       => __( 'Drop files here to upload', 'buddyboss' ),
+		'dropzone_document_message'       => sprintf( __( '<strong>Add Files</strong> Or drag and drop', 'buddyboss' ) ),
```

Comment:
Still wrong it should be something like this

```suggestion
		'dropzone_document_message'       => sprintf( '<strong>%s</strong> %s', __( 'Add Files', 'buddyboss' ), __( 'Or drag and drop', 'buddyboss' ),
```

Reply:
@chetansatasiya This is fixed now